### PR TITLE
Working with forms in Django

### PR DESCRIPTION
**Pull request by Anitaachu**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by Anitaachu** [comment]
This fixes #944

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR & Welcome 🎉  - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
The code works. I have made a few changes on the grammatical front as well. It is better to include screenshots/images rather than use other cloud-based services. Store the images in the same folder as the index.md and call the image accordingly instead of the cloud based image service you are using.

**Comment by Anitaachu** [comment]
Image changes done. Thank you.

File: articles/working-with-forms-in-django/index.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+</form>
+```
+
+Notice, we also added "**crispy"** in our code. Hence we are adding crispy to do the editing. Don't skip that.
+
+Once this is done properly, your form would look like this.
+
+ 
+
+![Register form 4 with crispy]()
```
Comment (lalith1403) [inline articles/working-with-forms-in-django/index.md:202]: I think you have missed the image file names. Can you insert the appropriate images at their respective places and LMK?

Comment (Anitaachu) [reply articles/working-with-forms-in-django/index.md]: Changes have been made. Thank you

**Comment by hectorkambow** [comment]
@Anitaachu  👋  Good afternoon Anita. 

Just giving your article a final review - Im noticing that (since this is your 1st article) we will need a short bio blurb and headshot image to add towards the bottom of all your articles. 

Please refer to any file in the author folder in the repo as a guide. 

👍  
Thank you 🚀  
cc: @lalith1403

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by Anitaachu** [comment]
Added bio blurb and headshot image. Thank you.

**Pull request merged**